src/ocean_data_assim/MOM_oda_driver.F90

```diff
@@ -332,7 +334,7 @@ subroutine init_oda(Time, G, GV, US, diag_CS, CS)
 
   h_neglect = set_h_neglect(GV, CS%answer_date, h_neglect_edge)
   call initialize_remapping(CS%remapCS, remap_scheme, om4_remap_via_sub_cells=om4_remap_via_sub_cells, &
-                            h_neglect=h_neglect, h_neglect_edge=h_neglect_edge)
+                            h_neglect=h_neglect, h_neglect_edge=h_neglect_edge, answer_date = CS%answer_date)
```

Comment:
No whitespace around assignment operators in functions: `answer_date=CS%answer_date`